autocomplete.go

```diff
@@ -40,6 +40,13 @@ func (a autocomplete) linuxAutocomplete(path string) (string, error) {
 	for _, subPath := range lastValidSplittedPath {
 		lastValidPath += "/" + subPath
 	}
+	if lastValidPath == "" {
+		lastValidPath = "/"
+	}
+
+	if !a.cmd.IsDir(lastValidPath) {
```

Comment:
I noticed that `a.cmd.IsDir()` and `isDir()` are functions that does the same logic. Could you just use one?